src/plugins/subsystems/drm/drm.sh

```diff
@@ -349,9 +372,65 @@ function get_supported_mode_per_connector()
   printf '%s\n' "$modes"
 }
 
+# Creates a binary EDID file, for each connected monitor, on the current folder.
```

Comment:
How about something like this:

Capture the EDID file from each connected monitors.